CHANGELOG.md

```diff
@@ -47,6 +47,7 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 
 ### Fixed
 - **cf-typography:** [PATCH] Fixed old variables removed from cf-core
+- **cf-forms:** [PATCH] Fixed a layout bug in btn-inside-input
```

Comment:
Trivial: It occurs to me that it's kinda superfluous to log bug fixes for the v4 conversion work, since the bugs never existed in a release.